### DISCUSSION TITLE
How can I get selected chain?

### DISCUSSION THREAD

**Discussion by thedevdwarf** [rainbow-me/rainbowkit#1481] [Q&A] [answered]
Hey,
I am using 3 chains in my project and I want to show different css for each chain how can I get the selected chain in this kit.  I searched but I couldn't find any information. 
Good day

**Comment by thedevdwarf** [comment, marked as answer]
I find a solution with this wagmi hook. Anyone looking for information on this -like me- could have a look here.
[https://wagmi.sh/react/hooks/useNetwork](url)

**Reply by Diogomartf** [reply]
But this requires the wallet to be connected, what about when isn't?

**Reply by DanielSinclair** [reply]
@Diogomartf In your dApp, you would typically default to a specific chain (i.e. Mainnet Ethereum), and then re-render when the wallet connects and a new chain is available (if that's the case).
